backend/src/govsg/middlewares/govsg-verification.middleware.ts

```diff
@@ -99,8 +100,11 @@ export const resendPasscodeCreationMessage = async (
     })
   }
   const { recipient } = govsgMessage
+  // flamingoDb numbers are prepended with +countrycode and have no whitespaces.
+  // recipient numbers of govsgMessage may not fulfil the above thus have to be preprocessed first.
+  const normalisedRecipient = PhoneNumberService.normalisePhoneNumber(recipient)
```

Comment:
Do we need to add this normalisation in the other instances where the `getApiClient` function is used (Looks like `govsg.class.ts` and `govsg-transactional.service.ts`)? I assume that this normalisation is always required?

Reply:
Those are created by the original coreteam and they are aware of the need for normalisation. If you do a quick search of `normalisePhoneNumber`, it should be already called in those places.

I never encountered this because I was operating from the webhook-side where the client ID is already given by WhatsApp.

Reply:
I agree, best not to touch the flamingo function. Thanks for your reply.